src/main/archetype/ui.frontend/src/main/webpack/resources/fonts/.gitignore

```diff
@@ -0,0 +1 @@
+
```

Comment:
I think this should probably be .gitkeep if you just want to include the empty folder. `.gitignore` seems wrong since we would want the fonts+images under source control.